dashboard/src/utils/constants/general.ts

```diff
@@ -7,3 +7,4 @@ export const DOCUMENTATION_URL = 'https://docs.kernelci.org/';
 // eslint-disable-next-line no-magic-numbers
 export const ItemsPerPageValues = [5, 10, 20, 30, 40, 50];
 export const DEFAULT_TIME_SEARCH = 7;
+export const defaultUrlLength = 12;
```

Comment:
`truncateBigText` has a default length of 50 characters, considering that `truncateUrl` has a default `endPathLength`of 20 we could up this to at least 30. Also, is this const really useful? Since we can pass the default value to the function argument itself (as it's already being done but with 50 instead of 12 or 30).

Reply:
When a pass no params, the git url don't truncate. So, when the value of url length was 12, I decide to keep this value and use in other url

Reply:
This might be good for the git url present in the Tree Details table. In other pages like Build and Test Details I believe they have enough space to justify increasing the number of characters